docs/false_positives.md

```diff
@@ -235,6 +236,84 @@ the analyzer better. Note that the compiler might generate the same code
 for the two implementations, so it might make sense to use only the more
 obvious one.
 
+## <a name="defensive-checks"></a> Defensive checks
+
+This section is an odd one because we describe a technique to reduce false
```

Comment:
Should the user really be interested in the deep reasoning behind why a particular subsection is part of this docfile, and not the other one?

Reply:
I think the main information here is that this is about how to reduce false negatives, not false positives. But I will remove the last sentence from this paragraph.